components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -36,36 +35,53 @@ The objects that you can manage from the Chef Infra Server are:
 - Policyfiles
 - PolicyGroup
 
+## Chef Automate WebUI Key
+
+Chef Automate uses the Web UI private key for signing requests sent to the Chef Infra Server. The key is subject to Chef server’s Secrets Management, and will be stored in `/etc/opscode/private-chef-secrets.json`. 
+
+The following example shows how the settings look when added to the configuration file:
```

Comment:
This example is not relevant to Automate. We can document the portion which talks about the command to find the web key from https://docs.chef.io/config_rb_manage/#web-ui-private-key